app/services/still_needs_help_service.rb

```diff
@@ -5,7 +5,26 @@ def self.must_show_still_needs_help_flow?(client)
 
   def self.trigger_still_needs_help_flow(client)
     client.triggered_still_needs_help_at = Time.now
-    client.tax_returns.where(status: %w[intake_ready intake_in_progress]).update(status: :file_not_filing)
+    client.tax_returns.where(status: may_need_help_statuses).update(status: :file_not_filing)
     client.save
+    send_still_need_help_notification(client)
+  end
+
+  def self.send_still_need_help_notification(client)
+    ClientMessagingService.send_system_message_to_all_opted_in_contact_methods(
+      client: client,
+      locale: client.intake.locale,
```

Comment:
I'd like to see a change someday where `send_system_message_to_all_opted_in_contact_methods` always grabs `locale` from `client.intake.locale`. Is there a scenario where that would be a mistake?

Not a blocker for this PR, naturally. Might be a nice refactor after this gets merged.

Related: I'd like to see `send_system_message_to_all_opted_in_contact_methods` use `I18n.with_locale(client.intake.locale) do ... end`, per https://translation.io/blog/rails-i18n-with-locale

Reply:
not sure, but that feels outside the scope of the ticket here.